### PR TITLE
API and Unit tests on Browserstack Test Observability using testng

### PR DESCRIPTION
**Pull request by Sahej-Jain**

### CODE REVIEW

File: .DS_Store
Comment (princebaretto99) [inline .DS_Store:1]: Delete this file

**Review by princebaretto99 — CHANGES_REQUESTED** [review]

File: .DS_Store
Comment (Sahej-Jain) [reply .DS_Store:1]: Done

**Review by princebaretto99 — DISMISSED** [review]

File: browserstack.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 userName: BROWSERSTACK_USERNAME
-accessKey: BROWSERSTACK_ACCESS_KEY
+accessKey: BROSWSERSTACK_ACCESS_KEY
```
Comment (ansel2000) [inline browserstack.yml]: Error is variable spelling, should be BROWSERSTACK_ACCESS_KEY

**Review by ansel2000 — CHANGES_REQUESTED** [review]

File: browserstack.yml (hunk 1 above)
Comment (Sahej-Jain) [reply browserstack.yml]: Have updated the variable spelling

**Review by princebaretto99 — APPROVED** [review]

**Review by ansel2000 — APPROVED** [review]

**Comment by samirans89** [comment]
please merge latest master into this branch before merging this branch into master

**Pull request merged**